app/brave_generated_resources.grd

```diff
@@ -333,6 +333,12 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
       <message name="IDS_SETTINGS_SPEEDREADER_SUB_LABEL" desc="The sub-label describing Speedreader">
         Articles automatically load in reader mode, saving you time
       </message>
+      <message name="IDS_SETTINGS_DE_AMP_LABEL" desc="The label to activate De-AMP feature">
```

Comment:
@petemill do you know why all these strings are in generated resources vs brave_strings? Is brave_strings auto-generated? Seems like we should have a dedicated strings file?

Reply:
not blocking on this PR